Src/BootCamp.Chapter/Checks.cs

```diff
@@ -14,28 +16,64 @@ namespace BootCamp.Chapter
     /// </summary>
     public static class Checks
     {
+        public const int IntegerError = -1;
+        public const string StringError = "-";
+        public const float FloatError = -1.0f;
+
         public static int PromptInt(string message)
         {
-            // To do: call your implementation. 
-            return 0;
+            Console.WriteLine(message);
+            string input = Console.ReadLine();
+            int variable;
+            bool num = int.TryParse(input, out variable);
```

Comment:
You can inline the variable declaration. Also please use more concrete names, variables can be literally said about anything.

```cs
bool isNumber = int.TryParse(input, out int number);
```